src/TimeSeries/encodings/timeseriespreprocessing.jl

```diff
@@ -0,0 +1,51 @@
+"""
+    TSPreprocessing() <: Encoding
+
+Encodes 'TimeSeriesRow's by normalizing the time-series values. The time-series can
+either be normalized by each variable or time-step.
+
+Encodes
+- 'TimeSeriesRow' -> 'TimeSeriesRow'
+"""
+
+struct TSPreprocessing <: Encoding
+    tfms
+end
+
+function TSPreprocessing()
+    base_tfms = [
+    ]
+    return TSPreprocessing(base_tfms)
+end
+
+function encodedblock(p::TSPreprocessing, block::TimeSeriesRow)
+    return block
+end
```

Comment:
If the format of the time series is changed by the encoding, this should return a different block

Reply:
No the format won't be changed, as I discussed with Brian earlier that different models might require different formats and so the encoding shouldn't depend on the model.